build_package.py

```diff
@@ -294,11 +294,11 @@ def build_win32_installer_package(variant, version):
     )
 
     # Copy the config file.
+    agent_json_path = make_path(agent_source_root, "config/agent.json")
     cat_files(
-        [make_path(agent_source_root, "config/agent.json")],
-        "agent_config.tmpl",
-        convert_newlines=True,
+        [agent_json_path], "agent_config.tmpl", convert_newlines=True,
     )
+    os.chmod(agent_json_path, int("640", 8))
```

Comment:
I would add a comment here that we are explicitly setting the "other" permission to zero to protect sensitive informatin in the config, or something like that.  I'd hate for someone who can't read chmod to change that without realizing how important it is.

Reply:
Agreed - there are also tests which catch this issue.